xarray/tutorial.py

```diff
@@ -77,8 +77,11 @@ def open_dataset(
     """
     try:
         import pooch
-    except ImportError:
-        raise ImportError("using the tutorial data requires pooch")
+    except ImportError as e:
+        raise ImportError(
+            "tutorial.open_dataset depends on pooch to download and manage datasets."
+            " To proceed please install it from PyPI or conda / conda-forge."
+        ) from e
```

Comment:
I'm not entirely sure this is better. People already used a package manager (of some form) to install xarray, so mentioning pypi/conda is not necessarily helpful.

Reply:
fair enough. Would dropping that be better: "To proceed please install pooch."

Reply:
yes!